src/main/java/Duke.java

```diff
@@ -1,10 +1,146 @@
+import java.lang.String;
+import java.util.Scanner;
+
 public class Duke {
+    public static final int MAX_INT = 100;
+    private static Task[] tasks = new Task[MAX_INT];
+
     public static void main(String[] args) {
+        // Level 4
+        displayDuke();
+        helloMessage();
+        commandsAvailable();
+
+        Scanner input = new Scanner(System.in);
+        int listIndex = 0;
```

Comment:
Consider elaborating your variable name to something more specific such as taskindex